draft-ietf-masque-connect-ip.md

```diff
@@ -863,7 +863,9 @@ forwarding can fail if the endpoint does not have a route for the destination
 address, or if it is configured to reject a destination prefix by policy, or if
 the MTU of the outgoing link is lower than the size of the packet to be
 forwarded. In such scenarios, IP proxying endpoints SHOULD use ICMP
-{{!ICMP=RFC0792}} {{!ICMPv6=RFC4443}} to signal the forwarding error to its peer.
+{{!ICMP=RFC0792}} {{!ICMPv6=RFC4443}} to signal the forwarding error to its peer
+by generating ICMP packets with corresponding IP headers and sending them
```

Comment:
We might need to spell out what "corresponding IP headers" means. The destination address is obvious, as that's the source address of the packet that caused the response. However, the source-address needs to be the CONNECT-IP endpoint's own address, which we do not guarantee is known, as there may be no assignment to it. If there is, we should use it, but if not, we need to use some identity (the machine's own default IPv6 address on some present interface?)

Reply:
This is clearly spelled out in the ICMP RFCs, no need for us to repeat that

Reply:
Discussed offline, removed "with corresponding IP headers" since that was more confusing than helpful